Cargo.toml

```diff
@@ -11,21 +11,36 @@ categories = ["authentication", "web-programming::http-server"]
 keywords = ["actix-web", "middleware", "authentication", "jwt", "keycloak"]
 license = "MIT"
 
+[[example]]
+name = "custom_claims"
+path = "examples/custom_claims.rs"
+required-features = ["paperclip_compat"]
+
+[[example]]
+name = "paperclip"
+path = "examples/paperclip.rs"
+required-features = ["paperclip_compat"]
+
+[[example]]
+name = "simple"
+path = "examples/simple.rs"
+required-features = ["paperclip_compat"]
+
 [dependencies]
-actix-web = { version = "4.4.0", default-features = false }
-chrono = { version = "0.4.31", features = ["serde"] }
-futures-util = { version = "0.3.29", default-features = false, features = ["std"] }
-log = "0.4.20"
-jsonwebtoken = "9.2.0"
-serde = { version = "1.0.193", features = ["derive"] }
+actix-web = { version = "4.9.0", default-features = false }
+chrono = { version = "0.4.38", features = ["serde"] }
+futures-util = { version = "0.3.30", default-features = false, features = ["std"] }
+log = "0.4.22"
+jsonwebtoken = "9.3.0"
+serde = { version = "1.0.210", features = ["derive"] }
 serde_json = "1.0.108"
-uuid = { version = "1.6.1", features = ["serde"] }
-paperclip = { version = "0.8.2", default-features = false, features = ["actix4"], optional = true }
+uuid = { version = "1.10.0", features = ["serde"] }
+paperclip = { version = "0.9.1", default-features = false, features = ["actix4", "paperclip-actix"], optional = true }
```

Comment:
I believe you do not need to explicitly add the `paperclip-actix` feature as it is enabled by the `actix4` one